RunningOrder/Shared/Components/Search/SearchBarSuggestions.swift

```diff
@@ -30,29 +36,38 @@ extension SearchBarSuggestions {
 
         var body: some View {
             ScrollView {
+//                Picker("Search Scope", selection: $logic.searchScope) {
+//                    ForEach(Scope.allCases, id: \.self) { scope in
+//                        Text(scope.title)
+//                            .tag(scope)
+//                    }
+//                }
+//                .labelsHidden()
+//                .pickerStyle(SegmentedPickerStyle())
+//                .padding()
+
```

Comment:
why is it commented ?

Reply:
because the picker doesn't work at the moment : it causes refresh cycle. I will try to investigate it today